plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/glue/DeltaLakeGlueMetastoreModule.java

```diff
@@ -42,12 +39,4 @@ protected void setup(Binder binder)
 
         install(new GlueMetastoreModule());
     }
-
-    @Provides
-    @Singleton
-    @HideNonDeltaLakeTables
-    public boolean provideHideNonDeltaLakeTables(DeltaLakeGlueMetastoreConfig glueMetastoreConfig)
-    {
-        return glueMetastoreConfig.isHideNonDeltaLakeTables();
```

Comment:
Is the config still used somewhere else?
Can we fix the problem without removing the config?

Reply:
> NOTE that the setting config.hide-non-delta-lake-tables it is still used in the context of the class io.trino.plugin.deltalake.metastore.glue.DeltaLakeGlueMetastoreTableFilterProvider